cdap-security/src/main/java/co/cask/cdap/security/authorization/RemoteAuthorizationEnforcer.java

```diff
@@ -99,6 +100,12 @@ public void enforce(EntityId entity, Principal principal, Action action) throws
     }
   }
 
+  @Override
+  public Set<? extends EntityId> isVisible(Set<? extends EntityId> entityIds, Principal principal) throws Exception {
+    // TODO: needs to be implemented
```

Comment:
we are not implementing it as of now since remote containers don't really need to do visibility check correct ?